test/transactions/transactionBytes.js

```diff
@@ -15,50 +15,250 @@
+	const defaultAppId = '1234213';
 
-	it('should be ok', () => {
-		(getTransactionBytes).should.be.ok();
+	describe('send transaction, type 0', () => {
+		let defaultTransaction;
+
+		beforeEach(() => {
+			defaultTransaction = {
+				type: 0,
+				amount: defaultAmount,
+				recipientId: defaultRecipient,
+				timestamp: defaultTimestamp,
+				asset: {},
+				senderPublicKey: defaultSenderPublicKey,
+				signature: defaultSignature,
+				id: defaultTransactionId,
+			};
+		});
+
+		it('should return Buffer of type 0 (send LSk) transaction', () => {
+			const expectedBuffer = Buffer.from('AKopAgBdA2qFjOifhESRdi64niv71QpKCg2mWOSyYoslsReuCQDOvKqNNBU96AMAAAAAAABhilSXUhLq2T34yIFlXGJVRLzo7XzN/m8IpC7s+xrevQUTB75QFLsFFhe694FdUPYhKecJGBkDYeXU3UeWVBsK', 'base64');
+			const transactionBytes = getTransactionBytes(defaultTransaction);
+
+			(transactionBytes).should.be.eql(expectedBuffer);
+		});
+
+		it('should return Buffer of type 0 (send LSk) with data', () => {
+			defaultTransaction.asset.data = 'Hello Lisk! Some data in here!...';
+			const expectedBuffer = Buffer.from('AKopAgBdA2qFjOifhESRdi64niv71QpKCg2mWOSyYoslsReuCQDOvKqNNBU96AMAAAAAAABIZWxsbyBMaXNrISBTb21lIGRhdGEgaW4gaGVyZSEuLi5hilSXUhLq2T34yIFlXGJVRLzo7XzN/m8IpC7s+xrevQUTB75QFLsFFhe694FdUPYhKecJGBkDYeXU3UeWVBsK', 'base64');
+			const transactionBytes = getTransactionBytes(defaultTransaction);
+
+			(transactionBytes).should.be.eql(expectedBuffer);
+		});
+
+		it('should throw on type 0 with too much data', () => {
+			defaultTransaction.asset.data = '0123456789012345678901234567890123456789012345678901234567890123456789';
+			(getTransactionBytes.bind(null, defaultTransaction)).should.throw('Transaction asset data exceeds size of 64.');
+		});
+
+		it('should return Buffer of transaction with second signature', () => {
+			defaultTransaction.signSignature = defaultSecondSignature;
+			const expectedBuffer = Buffer.from('AKopAgBdA2qFjOifhESRdi64niv71QpKCg2mWOSyYoslsReuCQDOvKqNNBU96AMAAAAAAABhilSXUhLq2T34yIFlXGJVRLzo7XzN/m8IpC7s+xrevQUTB75QFLsFFhe694FdUPYhKecJGBkDYeXU3UeWVBsKsAxK0ZiLyiRddENWYKJ4v+a/L176i9qW2Sf6v4tPb8/cspU/arrKoRnWiAmHpV3qDmNUvINmBStF+iMUVSICDw==', 'base64');
+			const transactionBytes = getTransactionBytes(defaultTransaction);
+
+			(transactionBytes).should.be.eql(expectedBuffer);
+		});
 	});
 
-	it('should be a function', () => {
-		(getTransactionBytes).should.be.type('function');
+	describe('signature transaction, type 1', () => {
+		const signatureTransaction = {
+			type: 1,
+			amount: defaultNoAmount,
+			fee: 500000000,
+			recipientId: null,
+			senderPublicKey: defaultSenderPublicKey,
+			timestamp: defaultTimestamp,
+			asset: { signature: { publicKey: defaultSenderSecondPublicKey } },
+			signature: defaultSignature,
+			id: defaultTransactionId,
+		};
+
+		it('should create correct transactionBytes from signature transaction', () => {
```

Comment:
`should return Buffer of type 1 (register second signature) transaction` would be more consistent with previous test